cocotb/share/makefiles/Makefile.sim

```diff
@@ -120,6 +120,11 @@ endif
 # Maintain backwards compatibility by supporting upper and lower case SIM variable
 SIM_LOWERCASE := $(shell echo $(SIM) | tr A-Z a-z)
 
+# Use riviera instead of aldec (depricated) 
+ifeq ($(SIM_LOWERCASE),aldec)    
+    SIM_LOWERCASE := riviera
+endif
```

Comment:
Can we put this in `Makefile.deprecations`?

Reply:
Yes! I was considering this and it seems a better place.

Reply:
Note that the first commit in this patch already did exactly that :)